lms/djangoapps/student_account/views.py

```diff
@@ -68,6 +68,7 @@ def login_and_registration_form(request, initial_mode="login"):
 
     """
     # Determine the URL to redirect to following login/registration/third_party_auth
+    # _local_server_get('/user_api/v1/account/registration/', request.session)
```

Comment:
what is this?

Reply:
Leftover from the debugging. Uncommented.